documentation/reference/test-api/testcontroller/openwindow.html

```diff
@@ -0,0 +1,2256 @@
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+    <li>
+		    
+		
+
+<span>Plugin API</span>
+
+		
+			<ul>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/plugin-api/browserprovider.html"  >BrowserProvider Interface</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/plugin-api/reporter.html"  >Reporter Interface</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+</ul>
+				</li>
+			</ul>
+		</div>
+	</nav>
+	
+
+	
+	
+	
+
+
+<main>
+	<article class="doc">
+		<div class="post-content">
+
+			<h1>t.openWindow method</h1>
+<p>Opens a new browser window. Returns the matching window descriptor.</p>
+<div class="highlight"><pre><code class="language-js" data-lang="js"><span class="nx">t</span><span class="p">.</span><span class="nx">openWindow</span><span class="p">(</span> <span class="nx">url</span> <span class="p">)</span> <span class="err">→</span> <span class="nb">Object</span>
+</code></pre></div><div class="table-wrapper"><table><thead><tr>
+<th>Parameter</th>
+<th>Type</th>
+<th>Description</th>
+</tr>
+</thead><tbody><tr>
+<td>url</td>
+<td>String</td>
+<td>The URL to open. Can be local or remote, absolute or relative.</td>
+</tr>
+</tbody></table></div>
+<p><strong>Example:</strong></p>
+<div class="highlight"><pre><code class="language-js" data-lang="js"><span class="k">import</span> <span class="p">{</span> <span class="nx">Selector</span> <span class="p">}</span> <span class="k">from</span> <span class="dl">'</span><span class="s1">testcafe</span><span class="dl">'</span><span class="p">;</span>
+
+<span class="nx">fixture</span> <span class="s2">`Example page`</span>
+    <span class="p">.</span><span class="nx">page</span> <span class="s2">`http://www.example.com/`</span><span class="p">;</span>
+
+<span class="nx">test</span><span class="p">(</span><span class="dl">'</span><span class="s1">Open the TestCafe website</span><span class="dl">'</span><span class="p">,</span> <span class="k">async</span> <span class="nx">t</span> <span class="o">=&gt;</span> <span class="p">{</span>
+    <span class="k">await</span> <span class="nx">t</span>
+        <span class="p">.</span><span class="nx">openWindow</span><span class="p">(</span><span class="dl">'</span><span class="s1">http://devexpress.github.io/testcafe</span><span class="dl">'</span><span class="p">)</span>
+        <span class="p">.</span><span class="nx">openWindow</span><span class="p">(</span><span class="dl">'</span><span class="s1">./documentation</span><span class="dl">'</span><span class="p">);</span>
+
+    <span class="kd">const</span> <span class="nx">localWebsite</span> <span class="o">=</span> <span class="k">await</span> <span class="nx">t</span><span class="p">.</span><span class="nx">openWindow</span><span class="p">(</span><span class="dl">'</span><span class="s1">file://path/to/my/website</span><span class="dl">'</span><span class="p">);</span>
```

Comment:
We don't check opening resources via file protocol.
So, it's better to change the `file://path/to/my/website` URL in example.